frontend/src/components/Permissions.tsx

```diff
@@ -95,28 +98,34 @@ const Permissions: React.FC<PermissionsProps> = ({
         />
       ) : (
         <>
-          <p className="text-sm font-medium mb-2">This app can:</p>
-          <div className="flex flex-col mb-2">
+          {/* {permissions.isolated && (
```

Comment:
This isn't needed anymore?

Reply:
It's not needed any more, now we have a card above titled "Isolated Balance"